public/ckeditor/samples/old/sample_posteddata.php

```diff
@@ -13,4 +13,4 @@
   For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
 -------------------------------------------------------------------------------------------
 
-</pre><div style="display:none"></body> */ include "assets/posteddata.php"; ?>
```

Comment:
Strange that the auto formatter decided to drop the close php script tag from this line... not sure that I like that.

Reply:
I'm not sure why that happened. I believe this is a vendor file and should be ignored (referencing your other comment) but it could also be that it was not on its own line or something of that nature. I'll make sure it didn't do it to any of the other files.

Reply:
Actually, I just checked and this PHP script ending tag isn't necessary since there's no HTML or anything else in the file (there's just a comment). So that matches the style of the rest of the codebase.

However, that doesn't change the fact that this file shouldn't be touched by the formatter so that portion needs to be fixed.